packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-soc-idp/go/integrationsteps.md

```diff
@@ -0,0 +1,163 @@
+## Integration steps
+
+### Step 1: Create the SDK client on application load
+
+When the user navigates to the home page and the application loads, create a new
+SDK Client object by calling the `NewClient` method.
+
+```go
+idx, err := idx.NewClient(
+      c.Okta.IDX.ClientID,
+      c.Okta.IDX.ClientSecret,
+      c.Okta.IDX.Issuer,
+      c.Okta.IDX.Scopes,
+      c.Okta.IDX.RedirectURI)
+if err != nil {
+    log.Fatalf("new client error: %+v", err)
+}
+```
+
+### Step 2: Build sign-in page and initialize client object
+
+Build a sign-in page that captures both the user's name and password.
+
+<div class="common-image-format common-image-format-vertical-margin">
+
+![A basic sign-in page in a Golang application](/img/oie-embedded-sdk/oie-embedded-sdk-go-use-case-basic-sign-on-page.png)
+
+</div>
+
+During page load, call the `Client's` `InitLogin` method. This method returns an object of type
+`LoginResponse` that is used to initate the sign-in process with Okta.  The object
+also contains a list of available social identity providers (IdPs) that is discussed in more detail in
+the next step.
+
+```go
+lr, err := s.idxClient.InitLogin(context.TODO())
+if err != nil {
+	log.Fatalf("Could not initalize login: %s", err.Error())
+}
+```
+
+### Step 3: Get available identity providers to show on sign-in page
```

Comment:
"Get the available identity providers to show on the sign-in page"

Reply:
Updated to "Get the available list of identity providers"